docs/source/models/intervention_models/crm_mgmt/vivarium_polypill.rst

```diff
@@ -125,6 +125,16 @@ with the regimen they have while on the polypill intervention.
 Simulants on the polypill intervention can move up the ladder to three 
 drugs at standard dose based on the usual treatment ramp shown :ref:`here <us_cvd_concept_model>`. 
 
+Starting polypill as three drugs at half dose was decided based on `British Formulary <https://vnras.com/wp-content/uploads/2020/03/BNF-78-1.pdf>`_ which is the source of dosing used in our SBP 
+treatment effect component (e.g., matches the other parts of this model). 
+
+The polypill includes: 
+- Amlodipine where the standard dose is 5 mg. Our polypill uses half standard dose or 2.5 mg. 
+- Losartan where the standard dose is 50 mg. Our polypill uses half standard dose or 25 mg. 
+- Hydrochlorothiazide is not used in isolation in the UK but is simialr to chlorthalidone in action and dosing, which has a standard dose of 25 mg. Our polypill uses half standard dose so that would map to 12.5 mg. 
```

Comment:
```suggestion
- Hydrochlorothiazide is not used in isolation in the UK but is similar to chlorthalidone in action and dosing, which has a standard dose of 25 mg. Our polypill uses half standard dose so that would map to 12.5 mg. 
```